### DISCUSSION TITLE
Habilitando Rota um index.html

### DISCUSSION THREAD

**Discussion by jucelinopinheiro** [balta-io/2811#28] [Q&A] [answered]
Bom dia! Na aula de “Arquivos estáticos”, como seria a forma correta no Asp.net 6, de apontar um index.html na pasta wwwroot, minha ideia é fazer um login via froms com html e javascript como fosse uma “SPA” hospedada junto da API e que a rota raiz “/” ficasse apontada para ele. 
Alterei no Program.cs e adicionei o “app.UseStaticFiles()”, mas gostaria de saber se estou no caminho certo.

//Program.cs
```
var builder = WebApplication.CreateBuilder(args);

//especificando como será usado o token onde está a chave
var key = Encoding.ASCII.GetBytes(Configuration.JwtKey);

builder
    .Services
    .AddControllers()
    .ConfigureApiBehaviorOptions(options =>
    {
        options.SuppressModelStateInvalidFilter = true;
    });

builder.Services.AddDbContext<BlogDataContext>();

var app = builder.Build();
app.UseStaticFiles();

//sempre na orderm Autentificação e depois autorização
app.UseAuthentication();
app.UseAuthorization();

app.MapControllers();
app.Run();
```

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @jucelinopinheiro, como vai?
Basta utilizar o `app.UseStaticFiles();` como comentei no curso... depois disso é só deixar os itens dentro da pasta `wwwroot` conforme comentei no curso também.

No caso do `index.html`, por ser um nome "padrão", basta acessar a raiz da aplicação (localhost:xxxx/) para visualizá-lo!
